orchestra/admin_views.py

```diff
@@ -28,8 +28,7 @@ def project_details(request, project_id):
 
 
 def _prettify_project_details(project, tasks_data, steps):
-    project['start_datetime'] = datetime.strptime(project['start_datetime'],
-                                                  '%Y-%m-%dT%H:%M:%S.%fZ')
+    project['start_datetime'] = parse_datetime(project['start_datetime'])
```

Comment:
Just confirming that parse_datetime will make this timezone-aware. Is there an option to tell it to use UTC?